.gitignore

```diff
@@ -38,3 +38,5 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+.vscode
+.vscode
```

Comment:
Don't commit this.

Reply:
this branch is based upon genericInputs, I have informed aditya will cherry pick changes once committed